test/e2e/tests/confirmations/helpers.ts

```diff
@@ -35,12 +35,6 @@ export function withRedesignConfirmationFixtures(
           metaMetricsId: 'fake-metrics-id',
           participateInMetaMetrics: true,
         })
-        .withPreferencesController({
```

Comment:
Is `withRedesignConfirmationFixtures` still needed?

Reply:
Good point, I renamed it to `withTransactionEnvelopeTypeFixtures`